currencyii/src/main/java/nl/tudelft/trustchain/currencyii/ui/bitcoin/BlockchainDownloadFragment.kt

```diff
@@ -24,8 +25,31 @@ class BlockchainDownloadFragment() : BaseFragment(R.layout.fragment_blockchain_d
 
     override fun onActivityCreated(savedInstanceState: Bundle?) {
         super.onActivityCreated(savedInstanceState)
+
+        // TODO: The routing is cleaner to do via the previously displayed fragment.
```

Comment:
I believe this to be fine for now.

Reply:
Indeed not the cleanest way to go back to the previous screen, but will do for now.